### ISSUE TITLE
Permissions requested seem overly broad

### ISSUE THREAD

**Issue by socketbox** [maxandersen/gdoc2adoc#21]
I'm not sure if this is an issue with Google Docs or Drive, or whether it's the result of an API call made within the script (other than the replacement of text, I couldn't find anything "destructive"), but the permissions that the user needs to grant the script seem overly broad for code that's intended to create one document from another (see image).

![broad_perms](https://user-images.githubusercontent.com/1593835/72496072-7ef9bd00-37f6-11ea-8397-7da716bf3395.png)

**Comment by maxandersen** [comment]
Yeah, I'm not sure how it figures out the permissions - I think this is an issue of it being a script embedded in the doc so it can basically do anything :/

If you know of a way to limit it i'm all ears!
